src/content/integrations/astro-vtbot.md

```diff
@@ -4,7 +4,6 @@ title: "astro-vtbot"
 description: "The 👜 Bag of Tricks ✨ for Astro's View Transitions"
 categories:
   - "css+ui"
```

Comment:
Keeping this around for now until @martrapp has time to add it as a keyword on npm

```suggestion
  - "css+ui"
  - "toolbar"
```